python/tools/wlr-libpython/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+pub fn add(left: usize, right: usize) -> usize {
+    left + right
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn it_works() {
+        let result = add(2, 2);
+        assert_eq!(result, 4);
+    }
+}
```

Comment:
This is the default code of the library. I would remove it and just add a comment about where the real logic of this crate is located.